vignettes/delay_distributions.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "A primer on working with delay distributions"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: resources/library.json
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{A primer on working with delay distributions}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette is intended to be guidance for working with probability distributions in R in the context of using delay distributions with _cfr_ to obtain delay-corrected estimates of disease severity.
+
+```{r setup}
+# load necessary packages
+library(cfr)
+
+# some distribution packages
+library(distributional)
+library(distcrete)
+```
+
+## A brief primer on distributions in R
+
+R provides rich and extensive support for representing and working with probability distributions, and this can be seen from the [CRAN probability distributions task view](https://cran.r-project.org/view=Distributions).
```

Comment:
Could be made clearer that a lot of the support for distributions is from packages. "R and its extension packages provide rich and extensive support..."?

Reply:
Added in https://github.com/epiverse-trace/cfr/pull/88/commits/96bcad74cf4e9dc0b1d3e4a30d06071c18485505